docs/src/languages/php/_index.md

```diff
@@ -22,212 +30,110 @@ Note that from PHP 7.1, the images use the Zend Thread Safe (ZTS) version of PHP
+{{< readFile file="src/registry/images/examples/full/php.app.yaml" highlight="yaml" location=".platform.app.yaml" >}}
 
-* On PHP 5.x, it's `/usr/sbin/php5-fpm`.
-* On PHP 7.0, it's `/usr/sbin/php-fpm7.0`.
-* On PHP 7.1, it's `/usr/sbin/php-fpm7.1-zts`.
-* On PHP 7.2, it's `/usr/sbin/php-fpm7.2-zts`.
-* On PHP 7.3, it's `/usr/sbin/php-fpm7.3-zts`.
-* On PHP 7.4, it's `/usr/sbin/php-fpm7.4-zts`.
-
-While you can call it manually that's generally not necessary.
-Note that PHP-FPM cannot run simultaneously along with another persistent process (such as ReactPHP or Amp).
-If you need both, they have to run in separate containers.
-
-## Expanded dependencies
-
-In addition to the standard `dependencies` format,
-it's also possible to specify alternative repositories for use by Composer.
-The standard format:
-
-```yaml
-dependencies:
-    php:
-        "platformsh/client": "2.x-dev"
-```
+### 2. Specify a dependency manager
 
-is equivalent to `composer require platform/client 2.x-dev`.
-You can also specify explicit `require` and `repositories` blocks:
+To manage PHP dependencies and libraries, use [Composer](https://getcomposer.org/).
+PHP containers use Composer 1.x by default. For more options, see [dependency management](#dependencies).
 
-```yaml
-dependencies:
-    php:
-        require:
-            "platformsh/client": "2.x-dev"
-        repositories:
-            - type: vcs
-              url: "git@github.com:platformsh/platformsh-client-php.git"
-```
+### 3. Serve your app
 
-That would install `platformsh/client` from the alternate repository specified, as a global dependency.
-In other words, it's equivalent to the following `composer.json` file:
-
-```json
-{
-    "repositories": [
-        {
-            "type": "vcs",
-            "url":  "git@github.com:platformsh/platformsh-client-php.git"
-        }
-    ],
-    "require": {
-        "platformsh/client": "2.x-dev"
-    }
-}
-```
+Web requests are processed as follows:
 
-That allows you to install a forked version of a global dependency from a custom repository.
+- Static files such as `.css` or `.jpg` files are served directly by the web server.
+- Requests are handled by the existing matching `.php` files defined in the `root` folder. Nonexistent files are handled by the front controller (`passthru`).
 
-## Build flavor
+To define from which URL path the content should be served, [set the `locations`](../../create-apps/app-reference.md#locations).
 
-PHP images use the `composer` build flavor by default,
-which runs `composer --no-ansi --no-interaction install --no-progress --prefer-dist --optimize-autoloader` if a `composer.json` file is detected.
+Note that all requests are sent to PHP.
 
-Note that by default, all PHP containers include the latest Composer 1.x release.
-If you wish to use Composer 2.x, add it as a `dependency`:
+In the below example, all requests made to the root of your site (`/`) are sent to the `public` directory:
```

Comment:
:clipboard: Avoid directional language: https://developers.google.com/style/translation?hl=en#general-dos-and-donts

```suggestion
In the following example, all requests made to the root of your site (`/`) are sent to the `public` directory:
```